org.ekstep.sunbirdcommonheader-1.5/editor/headerApp.js

```diff
@@ -59,6 +58,56 @@ angular.module('org.ekstep.sunbirdcommonheader:app', ["Scope.safeApply", "yaru22
     $scope.isFlagReviewer = false;
     $scope.editorEnv = "";
     $scope.showEditMeta = true;
+    $scope.contentCredits = [];
+    $scope.listLimit = 5;
+    /*
+    * Add owner details and update current count with new values.
+    */
+    $scope.updateOwnershipList = function(event, nodeData) {
```

Comment:
Where is the logic while editing the couser, textook getting all the Owsership list? 

Is this function is calling while editing as well(node.added event should trigger)?